js/index.js

```diff
@@ -40,3 +40,79 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navBar = document.querySelectorAll('a');
+let navB = Array.from(navBar);
+navB[0].textContent = 'Services'
+navB[1].textContent = 'Product'
+navB[2].textContent = 'Vision'
+navB[3].textContent = 'features'
+navB[4].textContent = 'About'
+navB[5].textContent = 'Contact'
+
+
+let naviGator = document.querySelector('nav');
+let add1 = document.createElement('a');
+add1.textContent = 'Home'
+add1.style.color = 'green'
+let add2 = document.createElement('a');
+add2.textContent = 'FAQ'
+add2.style.color = 'green'
+naviGator.prepend(add1)
+naviGator.appendChild(add2)
+
+navB.forEach(x => {
+  x.style.color = 'green';
+});
```

Comment:
When using an arrow function the `{}` are optional.
These lines could be shortened to
```js
navB.forEach(x => x.style.color = 'green');
```